run.sh

```diff
@@ -28,6 +29,8 @@ core config:
   dbpass: $DB_PASS
   dbname: $DB_NAME
   dbhost: db:3306
+  extra-php: |
+    define('WP_DEBUG', ${WP_DEBUG,,});
```

Comment:
Nothing wrong here, just a question for ya...

What does the `,,` do in `${WP_DEBUG,,}`? I've never seen that before. Can you explain? :smile: